01. 기본문제 풀이/02. 삼각형 판별하기/김보경/index.js

```diff
@@ -1,7 +1,13 @@
 function solution(a, b, c) {
-  let answer = 'YES';
+  let answer = "YES";
 
-  return answer;
+  let arr = [a, b, c];
+  let maxValeu = Math.max(...arr);
+  let result = arr.reduce((acc, el) => {
+    return el !== maxValeu ? acc + el : acc;
+  });
```

Comment:
삼항연산자 대신 전체 값을 더하고 maxValue값을 제거해주는 방법도 있을 거 같아요.
```suggestion
  let result = arr.reduce((acc, el) => acc + el, 0) - maxValue;
```